tests/test-util/test_transform.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import edward as ed
+import tensorflow as tf
+from collections import namedtuple
+from edward.models import (Beta, Dirichlet, Gamma, MultivariateNormalDiag,
+                           Normal, PointMass, TransformedDistribution)
+from tensorflow.contrib.distributions import bijectors
+
+
+class test_transform_class(tf.test.TestCase):
+
+  def test_args(self):
+    with self.test_session():
+      x = Normal(0.0, 1.0)
+      y = ed.transform(x, bijectors.Softplus())
+      y.eval()
+
+  def test_kwargs(self):
+    with self.test_session():
+      x = Normal(0.0, 1.0)
+      y = ed.transform(x, bijector=bijectors.Softplus())
+      y.eval()
```

Comment:
Can you check that the value is non-negative? For example, you might start with a normal distribution with mean of -100.0, standard deviation 0.1. Then assert that 10 draws from the transformed distribution are all non-negative.

Same goes for the other draws.

Reply:
Good idea, I'll do that.

More generally, do you have any opinion regarding testing of probabilistic components? I'll just do a single sample check for now, but more would be better. It might be worth having a rule of thumb guideline to apply across the codebase (say, 20 samples). Even better would be an importable global variable that can be set when calling the test runner.